openedx/frontend-app-authoring
# feat: add to collection menu item functionality [FC-0062]

## Description

Clicking on `Add to collection` menu item will display sidebar with `Manage` tab open and `Tags` section collapsed for better visibility of collections section. Also, refactors library context to replace three separate state variables with single object type

## Supporting information

* Implements https://github.com/openedx/frontend-app-authoring/issues/1394
* `Private-ref`: [FAL-3848](https://tasks.opencraft.com/browse/FAL-3848)

## Testing instructions

* Click on `Add to Collection` menu item in components card as well as in the sidebar, it should open up sidebar with Manage tab active and tags section collapsed.gg
